librz/core/cmd_analysis.c

```diff
@@ -757,16 +757,16 @@ static int cmpname(const void *_a, const void *_b) {
 static int cmpsize(const void *a, const void *b) {
 	ut64 sa = (int)rz_analysis_function_linear_size((RzAnalysisFunction *)a);
 	ut64 sb = (int)rz_analysis_function_linear_size((RzAnalysisFunction *)b);
-	return (sa > sb) ? -1 : (sa < sb) ? 1
-					  : 0;
+	return (sa > sb) ? 1 : (sa < sb) ? 1
```

Comment:
I am not sure this change is right. Why it does return `1` in both cases? This is not how comparison callback works...